packages/amplify-frontend-ios/lib/frontend-config-creator.js

```diff
@@ -43,12 +44,21 @@ function getSrcDir(context) {
   return path.join(projectPath);
 }
 
-function createAmplifyConfig(context, amplifyResources) {
-  const srcDirPath = getSrcDir(context);
+function createAmplifyConfig(context) {
```

Comment:
This is somewhat similar to packages/amplify-frontend-android/lib/frontend-config-creator.js  line 50